src/components/Spinner.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div v-if="start">
+    <div class="lds-ring">
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: ["start"],
```

Comment:
Spinner shouldn't have the `start` prop or the `v-if`. We are going to control it in the parent